src/main/java/frc/lib/team3015/subsystem/SubsystemFault.java

```diff
@@ -0,0 +1,68 @@
+package frc.lib.team3015.subsystem;
+
+import edu.wpi.first.wpilibj.Timer;
+import frc.lib.team6328.util.Alert;
+
+import java.util.Objects;
+
+public class SubsystemFault{
+  public final String description;
+  public final double timestamp;
+  public final boolean isWarning;
+  public final boolean sticky;
+  public SubsystemFault(String description, boolean isWarning) {
+    this(description, isWarning, false);
+  }
+
+  public SubsystemFault(String description) {
+    this(description, false);
+  }
+
+  // what does sticky mean specifically?
```

Comment:
Sticky means that the fault should remain visible. In terms of the FaultReporter, it would remain visible until the code restarts. If sticky is false, the fault expires after 10 seconds. From an alert perspective, I think it is fine to treat all faults as sticky rather than disabling alerts for non-sticky faults after a period of time.